common/utils/openebs/upgrade/test_prerequisites.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Update repositories cache and install subversion package
+  apt:
+    name: subversion
+    update_cache: yes
+
+- name: check for subvsersion installation.
+  shell: dpkg-query -l 'subversion'
+  args:
+    executable: /bin/bash
+  register: version
+  failed_when: "'no packages found' in version.stdout"
+
+
+# Get the openebs upgrade artifacts form openebs repository.
+- subversion:
+    repo: "{{ git_repo }}{{ version_tag }}"
+    dest: /openebs_upgrade/
+
+- name: Check presence required openebs_upgrade files
```

Comment:
@vibhor995 can you give the proper task name to understand?

Reply:
@nsathyaseelan sure